ecs-cli/integ/cmd/compose.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2015-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/integ"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/integ/stdout"
+)
+
+// A Project is the configuration needed to create an ECS Service.
+type Project struct {
+	Name       string
+	ConfigName string
+}
+
+// NewProject creates a new Project for an ECS Service.
+func NewProject(name string, configName string) *Project {
+	return &Project{
+		Name:       name,
+		ConfigName: configName,
+	}
+}
+
+// TestServiceUp runs `ecs-cli compose service up` for a project.
+func TestServiceUp(t *testing.T, p *Project) {
+	// Given
+	args := []string{
+		"compose",
+		"--project-name",
+		p.Name,
+		"service",
+		"up",
+		"--create-log-groups",
```

Comment:
What's the behavior if these log groups already exist? I believe it outputs a warning (vs. failing) - will this interfere with later `stdout` assertions?